lspserver/src/fstar.ts

```diff
@@ -29,6 +30,7 @@ export class FStar {
 			// Indicates whether the F* process supports full-buffer mode
 			public supportsFullBuffer: boolean,
 			public lax: boolean,
+			public fstarIdeVersion: integer = 3//default F* ide version is 3
```

Comment:
```suggestion
			public fstarIdeVersion: number = 3, //default F* ide version is 3
```

`number` is the built-in typescript type